web/css/codeworld-cm.css

```diff
@@ -85,26 +85,27 @@ li.CodeMirror-hint-active {
     color: #555;
 }
 
-.CodeMirror-hover-tooltip {
+.CodeMirror-hover-tooltip, .hint-description {
     background-color: #ffe;
-
     border-radius: 0.5em;
-    border: 1px solid #888;
+    border: 0.05em solid #888;
```

Comment:
I still like `1px` for borders, as they look cleaner that way.  It's just significant distances that are better measured in `em`.

Reply:
ok